security/container_security/security-container-signature.adoc

```diff
@@ -0,0 +1,17 @@
+[id="security-container-signature"]
```

Comment:
Make sure the top of the assembly fits the requirements here: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata